src/posix/main.c

```diff
@@ -153,37 +129,12 @@ static void PrintUsage(const char *aProgramName, FILE *aStream, int aExitCode)
             "    -d  --debug-level             Debug level of logging.\n"
             "    -h  --help                    Display this usage information.\n"
             "    -I  --interface-name name     Thread network interface name.\n"
+            "        --radio-url url           Radio url.\n"
```

Comment:
if posix need to parse protocols, radio dev, args from the url, is it better to set url as a required argument instead of option "--radio-url"?